src/Chapter06/Listing06.26.DefiningAPrimaryConstructor.cs

```diff
@@ -0,0 +1,66 @@
+﻿namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_26;
+
+#if NET8_0_OR_GREATER
+public class Program
+{
+    public static void Main()
+    {
+        Employee employee;
+        employee = new("Inigo", "Montoya");
+        employee.Salary = "Too Little";
+
+        System.Console.WriteLine(
+            "{0} {1}: {2}",
+            employee.FirstName,
+            employee.LastName,
+            employee.Salary);
```

Comment:
I personally favor interpolation here as I think it reads nicer.
```suggestion
        System.Console.WriteLine(
            $"{employee.FirstName} {employee.LastName}: {employee.Salary}");
```